packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -137,6 +137,7 @@ export class TestObjectProvider {
     private _urlResolver: IUrlResolver | undefined;
     private _logger: ITelemetryBaseLogger | undefined;
     private readonly _documentIdStrategy: IDocumentIdStrategy;
+    private _usedDocumentId: boolean;
```

Comment:
`usedDocumentId` does not seem like the right name for this. We are limiting the number of documents or number of create container calls to 1. So, the name should be something like documentCreated or containerCreated.
You should also add a comment here about this limit.